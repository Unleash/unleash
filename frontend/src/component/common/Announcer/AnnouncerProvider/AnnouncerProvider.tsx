import React, { ReactElement, useMemo, useState, ReactNode } from 'react';
import { AnnouncerContext } from '../AnnouncerContext/AnnouncerContext';
import { AnnouncerElement } from 'component/common/Announcer/AnnouncerElement/AnnouncerElement';

interface IAnnouncerProviderProps {
    children: ReactNode;
}

export const AnnouncerProvider = ({
    children,
}: IAnnouncerProviderProps): ReactElement => {
    const [announcement, setAnnouncement] = useState<string>();

    const value = useMemo(
        () => ({
            setAnnouncement,
        }),
        [setAnnouncement]
    );

    return (
        <AnnouncerContext.Provider value={value}>
            {children}
            <AnnouncerElement announcement={announcement} />
        </AnnouncerContext.Provider>
    );
};
