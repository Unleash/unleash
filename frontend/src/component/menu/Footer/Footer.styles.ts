import { makeStyles } from 'tss-react/mui';

export const useStyles = makeStyles()(theme => ({
    footer: {
        padding: '2rem 4rem',
        width: '100%',
        flexGrow: 1,
        zIndex: 100,
        backgroundColor: theme.palette.footerBackground,
    },
    list: {
        padding: 0,
        margin: 0,
    },
    listItem: {
        padding: 0,
        margin: 0,
        '& a': {
            textDecoration: 'none',
            color: theme.palette.text.primary,
        },
    },
}));
