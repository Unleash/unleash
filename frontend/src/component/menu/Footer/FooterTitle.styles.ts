import { makeStyles } from 'tss-react/mui';

export const useStyles = makeStyles()(theme => ({
    title: {
        all: 'unset',
        display: 'block',
        margin: '1rem 0',
        fontSize: '1rem',
        fontWeight: theme.fontWeight.bold,
    },
}));
