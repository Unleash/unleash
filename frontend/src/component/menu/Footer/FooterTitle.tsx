import { ReactNode } from 'react';
import { useStyles } from 'component/menu/Footer/FooterTitle.styles';

interface IFooterTitleProps {
    children: ReactNode;
}

export const FooterTitle = ({ children }: IFooterTitleProps) => {
    const { classes: styles } = useStyles();

    return <h2 className={styles.title}>{children}</h2>;
};
