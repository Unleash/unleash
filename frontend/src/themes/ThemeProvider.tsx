import React, { FC } from 'react';
import { CssBaseline, ThemeProvider as MuiThemeProvider } from '@mui/material';

import createCache from '@emotion/cache';
import { CacheProvider } from '@emotion/react';
import { useThemeMode } from 'hooks/useThemeMode';

export const muiCache = createCache({
    key: 'mui',
    prepend: true,
});

export const ThemeProvider: FC = ({ children }) => {
    const { resolveTheme } = useThemeMode();

    return (
        <CacheProvider value={muiCache}>
            <MuiThemeProvider theme={resolveTheme()}>
                <CssBaseline />
                {children}
            </MuiThemeProvider>
        </CacheProvider>
    );
};
