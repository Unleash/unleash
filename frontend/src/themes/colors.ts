/**
 * IMPORTANT: Use full color palette ONLY in theme. Don't import colors directly from this file.
 *
 * @see https://www.figma.com/file/qdwpPfuitJUNinm6mvmCmG/Unleash-application?node-id=7175%3A44590
 */
export const colors = {
    white: '#FFFFFF',
    black: '#000000',
    grey: {
        900: '#202021',
        800: '#6E6E70',
        700: '#78787A',
        600: '#9F9FA1',
        500: '#BDBDBF',
        400: '#E1E1E3',
        300: '#EAEAED',
        200: '#F2F2F5',
        100: '#F7F7FA',
        50: '#FFFFFF',
    },
    purple: {
        900: '#615BC2',
        800: '#6C65E5',
        700: '#817AFE',
        600: '#8C87EB',
        500: '#9D98EE',
        400: '#ADA9F1',
        300: '#BEBBF3',
        200: '#CECCF6',
        100: '#E4E3F9',
        50: '#F1F0FC',
    },
    red: {
        900: '#A6000E',
        800: '#D11525',
        700: '#D93644',
        600: '#E04C59',
        500: '#E04C59',
        400: '#F0616D',
        300: '#FEB0B7',
        200: '#FFD4D8',
        100: '#FFE5E7',
        50: '#FFF2F3',
    },
    orange: {
        900: '#B35300',
        800: '#D76500',
        700: '#F28D00',
        600: '#FFAA33',
        500: '#FFC46F',
        400: '#FFCF8B',
        300: '#FFD9A3',
        200: '#FFEACC',
        100: '#FFF4E5',
        50: '#FFFCF5',
    },
    green: {
        900: '#305200',
        800: '#3B6600',
        700: '#4D8400',
        600: '#68A611',
        500: '#7FB435',
        400: '#99C35D',
        300: '#B0D182',
        200: '#CFE5AE',
        100: '#E4F0D3',
        50: '#F4FAEB',
    },
    blue: {
        900: '#163E59',
        800: '#0060A1',
        700: '#0071BD',
        600: '#007ACA',
        500: '#0087E0',
        400: '#1C98EB',
        300: '#5BB4F0',
        200: '#96D2FA',
        100: '#DCEEFA',
        50: '#EBF7FF',
    },
    darkblue: {
        1000: '#091826',
        900: '#041c32',
        800: '#1d3247',
        700: '#28415c',
        600: '#1f3751',
        500: '#0e2840',
    },
} as const;
