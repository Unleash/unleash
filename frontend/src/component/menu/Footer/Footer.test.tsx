import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Footer from './Footer';
import { ThemeProvider } from 'themes/ThemeProvider';
import { AnnouncerProvider } from 'component/common/Announcer/AnnouncerProvider/AnnouncerProvider';

test('should render DrawerMenu', () => {
    const tree = renderer.create(
        <ThemeProvider>
            <AnnouncerProvider>
                <MemoryRouter>
                    <Footer />
                </MemoryRouter>
            </AnnouncerProvider>
        </ThemeProvider>
    );

    expect(tree).toMatchSnapshot();
});

test('should render DrawerMenu with "features" selected', () => {
    const tree = renderer.create(
        <ThemeProvider>
            <AnnouncerProvider>
                <MemoryRouter initialEntries={['/features']}>
                    <Footer />
                </MemoryRouter>
            </AnnouncerProvider>
        </ThemeProvider>
    );

    expect(tree).toMatchSnapshot();
});
