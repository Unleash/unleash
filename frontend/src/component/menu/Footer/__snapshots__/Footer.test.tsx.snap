// Vitest Snapshot v1

exports[`should render DrawerMenu 1`] = `
[
  <footer
    className="css-16d7ppb-footer"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 css-16chest-MuiGrid-root"
      style={
        {
          "marginBottom": 0,
        }
      }
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-g8shpt-MuiGrid-root"
      >
        <section
          title="API details"
        >
          <h2
            className="css-cxwipi-title"
          >
            Unleash 3.x
             
          </h2>
          <br />
          <small>
            The enterprise ready feature toggle service.
          </small>
          <br />
        </section>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-auto css-16ccgsq-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-7 css-3f6ufr-MuiGrid-root"
        >
          <div
            className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <section
              title="Unleash SDK"
            >
              <h2
                className="css-cxwipi-title"
              >
                Server SDKs
              </h2>
              <ul
                className="MuiList-root MuiList-padding MuiList-dense css-e6vijf-MuiList-root-list"
              >
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/node_sdk"
                        target="_blank"
                      >
                        Node.js
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/java_sdk"
                        target="_blank"
                      >
                        Java
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/go_sdk"
                        target="_blank"
                      >
                        Go
                      </a>
                    </span>
                  </div>
                </li>
                 
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/ruby_sdk"
                        target="_blank"
                      >
                        Ruby
                      </a>
                    </span>
                  </div>
                </li>
                 
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/python_sdk"
                        target="_blank"
                      >
                        Python
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/dot_net_sdk"
                        target="_blank"
                      >
                        .NET
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/php_sdk"
                        target="_blank"
                      >
                        PHP
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks"
                        target="_blank"
                      >
                        All SDKs
                      </a>
                    </span>
                  </div>
                </li>
              </ul>
            </section>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <section
              title="Unleash SDK"
            >
              <h2
                className="css-cxwipi-title"
              >
                Frontend SDKs
              </h2>
              <ul
                className="MuiList-root MuiList-padding MuiList-dense css-e6vijf-MuiList-root-list"
              >
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/unleash-proxy"
                        target="_blank"
                      >
                        Unleash Proxy
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/proxy-javascript"
                        target="_blank"
                      >
                        JavaScript SDK
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/proxy-react"
                        target="_blank"
                      >
                        React SDK
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/proxy-ios"
                        target="_blank"
                      >
                        iOS SDK
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/android_proxy_sdk"
                        target="_blank"
                      >
                        Android SDK
                      </a>
                    </span>
                  </div>
                </li>
              </ul>
            </section>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <section>
              <h2
                className="css-cxwipi-title"
              >
                About
              </h2>
              <ul
                className="MuiList-root MuiList-padding MuiList-dense css-e6vijf-MuiList-root-list"
              >
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://www.getunleash.io/"
                        target="_blank"
                      >
                        getunleash.io
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://twitter.com/getunleash"
                        target="_blank"
                      >
                        Twitter
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://www.linkedin.com/company/getunleash"
                        target="_blank"
                      >
                        LinkedIn
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://github.com/Unleash/unleash"
                        target="_blank"
                      >
                        GitHub
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://slack.unleash.run"
                        target="_blank"
                      >
                        Slack Community
                      </a>
                    </span>
                  </div>
                </li>
              </ul>
            </section>
          </div>
        </div>
      </div>
    </div>
  </footer>,
  <div
    aria-atomic={true}
    aria-live="polite"
    className="css-i8rqz1-container"
    data-testid="ANNOUNCER_ELEMENT_TEST_ID"
    role="status"
  />,
]
`;

exports[`should render DrawerMenu with "features" selected 1`] = `
[
  <footer
    className="css-16d7ppb-footer"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 css-16chest-MuiGrid-root"
      style={
        {
          "marginBottom": 0,
        }
      }
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-g8shpt-MuiGrid-root"
      >
        <section
          title="API details"
        >
          <h2
            className="css-cxwipi-title"
          >
            Unleash 3.x
             
          </h2>
          <br />
          <small>
            The enterprise ready feature toggle service.
          </small>
          <br />
        </section>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-auto css-16ccgsq-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-7 css-3f6ufr-MuiGrid-root"
        >
          <div
            className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <section
              title="Unleash SDK"
            >
              <h2
                className="css-cxwipi-title"
              >
                Server SDKs
              </h2>
              <ul
                className="MuiList-root MuiList-padding MuiList-dense css-e6vijf-MuiList-root-list"
              >
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/node_sdk"
                        target="_blank"
                      >
                        Node.js
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/java_sdk"
                        target="_blank"
                      >
                        Java
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/go_sdk"
                        target="_blank"
                      >
                        Go
                      </a>
                    </span>
                  </div>
                </li>
                 
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/ruby_sdk"
                        target="_blank"
                      >
                        Ruby
                      </a>
                    </span>
                  </div>
                </li>
                 
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/python_sdk"
                        target="_blank"
                      >
                        Python
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/dot_net_sdk"
                        target="_blank"
                      >
                        .NET
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/php_sdk"
                        target="_blank"
                      >
                        PHP
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks"
                        target="_blank"
                      >
                        All SDKs
                      </a>
                    </span>
                  </div>
                </li>
              </ul>
            </section>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <section
              title="Unleash SDK"
            >
              <h2
                className="css-cxwipi-title"
              >
                Frontend SDKs
              </h2>
              <ul
                className="MuiList-root MuiList-padding MuiList-dense css-e6vijf-MuiList-root-list"
              >
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/unleash-proxy"
                        target="_blank"
                      >
                        Unleash Proxy
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/proxy-javascript"
                        target="_blank"
                      >
                        JavaScript SDK
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/proxy-react"
                        target="_blank"
                      >
                        React SDK
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/proxy-ios"
                        target="_blank"
                      >
                        iOS SDK
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://docs.getunleash.io/sdks/android_proxy_sdk"
                        target="_blank"
                      >
                        Android SDK
                      </a>
                    </span>
                  </div>
                </li>
              </ul>
            </section>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <section>
              <h2
                className="css-cxwipi-title"
              >
                About
              </h2>
              <ul
                className="MuiList-root MuiList-padding MuiList-dense css-e6vijf-MuiList-root-list"
              >
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://www.getunleash.io/"
                        target="_blank"
                      >
                        getunleash.io
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://twitter.com/getunleash"
                        target="_blank"
                      >
                        Twitter
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://www.linkedin.com/company/getunleash"
                        target="_blank"
                      >
                        LinkedIn
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://github.com/Unleash/unleash"
                        target="_blank"
                      >
                        GitHub
                      </a>
                    </span>
                  </div>
                </li>
                <li
                  className="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-17d9xc9-MuiListItem-root-listItem"
                  disabled={false}
                >
                  <div
                    className="MuiListItemText-root MuiListItemText-dense css-tlelie-MuiListItemText-root"
                  >
                    <span
                      className="MuiTypography-root MuiTypography-body2 MuiListItemText-primary css-1nhzl8d-MuiTypography-root"
                    >
                      <a
                        href="https://slack.unleash.run"
                        target="_blank"
                      >
                        Slack Community
                      </a>
                    </span>
                  </div>
                </li>
              </ul>
            </section>
          </div>
        </div>
      </div>
    </div>
  </footer>,
  <div
    aria-atomic={true}
    aria-live="polite"
    className="css-i8rqz1-container"
    data-testid="ANNOUNCER_ELEMENT_TEST_ID"
    role="status"
  />,
]
`;
